exasol_script_languages_container_ci_setup/lib/aws/wrapper/secretsmanager_service.py

```diff
@@ -0,0 +1,22 @@
+import dataclasses
+from typing import Any, Dict, Callable
+
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.common import ARN, PhysicalResourceId
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.secretsmanager import Secret
+
+
+class SecretsManagerService:
+    def __init__(self, boto_client):
+        self._boto_client = boto_client
+
+    @property
+    def boto_client(self) -> Any:
+        return self._boto_client
```

Comment:
You are returning `Any` so is it relevant that it is named `boto_client` rather than just `client`?